packages/react-components/package.json

```diff
@@ -16,11 +16,12 @@
   "scripts": {
     "build": "just-scripts build",
     "bundle": "just-scripts bundle",
+    "bundle:storybook": "cross-env NODE_OPTIONS=--max-old-space-size=4096 just-scripts storybook:build",
     "clean": "just-scripts clean",
     "code-style": "just-scripts code-style",
     "just": "just-scripts",
     "lint": "just-scripts lint",
-    "start": "just-scripts dev:storybook"
+    "start": "cross-env NODE_OPTIONS=--max-old-space-size=4096 just-scripts dev:storybook"
```

Comment:
**q:** why do we use just for convergence? based on my approved RFC we should use storybook directly

Reply:
This is just adding more memory to the existing command in case it's being used. You can feel free to remove this script later once you get to that point in the convergence work.